app/src/androidTest/java/com/github/polypoly/app/game/PendingGameTest.kt

```diff
@@ -137,10 +140,10 @@ class PendingGameTest {
         val pendingGame = PendingGame(testUser, GameMode.RICHEST_PLAYER, testMinNumberPlayers, testMaxNumberPlayers,
             testDuration, emptyList(), testInitialBalance, testName, testCode)
 
-        val u1 = User(42042043, "test_user1", "", emptySkin, zeroStats)
-        val u2 = User(42042044, "test_user2", "", emptySkin, zeroStats)
-        val u3 = User(42042045, "test_user3", "", emptySkin, zeroStats)
-        val u4 = User(42042046, "test_user4", "", emptySkin, zeroStats)
+        val u1 = User(42042043, "test_user1", "", emptySkin, zeroStats, listOf(), mutableListOf())
+        val u2 = User(42042044, "test_user2", "", emptySkin, zeroStats, listOf(), mutableListOf())
+        val u3 = User(42042045, "test_user3", "", emptySkin, zeroStats, listOf(), mutableListOf())
+        val u4 = User(42042046, "test_user4", "", emptySkin, zeroStats, listOf(), mutableListOf())
 
```

Comment:
This initialization is redundant, so it would be better to add them as class members testUser2, testUser3, testUser4 and testUser5 instead